integration-tests/resteasy-reactive-kotlin/src/main/kotlin/io/quarkus/it/resteasy/reactive/kotlin/CountriesEndpoint.kt

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.it.resteasy.reactive.kotlin
+
+import kotlinx.coroutines.delay
+import org.eclipse.microprofile.rest.client.inject.RestClient
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+
+@Path("country")
+class CountriesEndpoint(@RestClient private val countriesService: CountriesService) {
+
+    @GET
+    @Path("/name/{name}")
+    suspend fun byName(name: String): Set<Country> {
+        delay(50)
```

Comment:
This is really testing the server side and not the client side, as the delay happens before any client code is hit.

Reply:
Good point

Reply:
Moved `delay` after the client call